package.json

```diff
@@ -53,7 +53,7 @@
     "@types/react-test-renderer": "^16.9.1",
     "@types/sanitize-filename": "^1.6.3",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^12.8.1",
+    "cypress": "^13.6.1",
```

Comment:
Should we include major version bump in this PR without testing?

Reply:
Some time back I looked through the Cypress 12 -> 13 migration guide and none of the breaking changes were affecting us. Integration tests are still failing with both versions, though.

Reply:
Should we be on the same version as FTR, cause we have some tests here and some basic tests in FTR. Wouldn't it be difficult for developers to maintain syntax of two different versions? 

I'm fine with either, just putting my perspective for long term easiness.

Reply:
There's already an open issue to migrate the FTR tests to 12: #957. The compatible route is to stay on 12, but in the 12->13 case specifically, it should bring extra performance gains under the hood without any code difference overhead.

Reply:
Same thoughts here, both are fine, but seems like there's no direct and necessary benefit from this version bump for this PR, we could do it with separate PR or later in a step by step manner.

Reply:
> There's already an open issue to migrate the FTR tests to 12: https://github.com/opensearch-project/dashboards-observability/issues/957. The compatible route is to stay on 12, but in the 12->13 case specifically, it should bring extra performance gains under the hood without any code difference overhead.

Thanks for the reference @Swiddis. For some additional context, that issue was originally kicked off by the security team, security frontend needs the cypress version >11 for testing external auth. However, since the FTR has its own overwrite commands with basic auth only (such as `cy.visit()`, `cy.request()`) and tons of tests were relying on these commands, it was hard to do the refactor work. But later on the security team introduced the Cypress 13 in their own repo. In addition, compare to Cypress 9, newer version does have a huge performance improvement.

At the same time, I agree with @mengweieric and @ps48. I can remove the version bump in this one then and maybe we can do the version bump as a follow-up. However, from my perspective, I think it is fine to stay in Cypress 12 then, and we don't need to downgrade our version to match with FTR. Cuz we are using this dependency only for our own workflow, and we don't have any syntax conflict at this point.

Reply:
The version bump of this PR has been removed.